obs-packaging/get_pkg_data.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+#Copyright (c) 2018 Intel Corporation
+#
+#SPDX-License-Identifier: Apache-2.0
+#
+
+[ -z "${DEBUG}" ] || set -o xtrace
+set -o errexit
+set -o nounset
+set -o pipefail
+
+source ../scripts/lib.sh
+
+pkg=$1
+git_reference="1.1.0"
+case "${pkg}" in
+	qemu-lite)
+		pkg_repository="https://github.com/kata-containers/qemu.git"
+		git_reference=$(get_from_kata_deps assets.hypervisor.qemu-lite.branch "${git_reference}")
+		;;
+	qemu-vanilla)
+		pkg_repository="https://github.com/kata-containers/qemu.git"
+		git_reference=$(get_from_kata_deps assets.hypervisor.qemu.version "${git_reference}")
+		;;
+	runtime)
+		pkg_repository="https://github.com/kata-containers/runtime.git"
+		;;
+	shim)
+		pkg_repository="https://github.com/kata-containers/shim.git"
+		;;
+	proxy)
+		pkg_repository="https://github.com/kata-containers/proxy.git"
+		;;
+	ksm-throttler)
+		pkg_repository="https://github.com/kata-containers/ksm-throttler.git"
+		;;
+	kernel)
+		pkg_repository="https://github.com/kata-containers/packaging.git"
+		git_reference+="-kernel-config"
+		;;
+	image)
+		pkg_repository="https://github.com/kata-containers/agent.git"
+		#Image is built not git refernce for this
```

Comment:
Maybe drop this comment as the same applies to shim, proxy, runtime and throttler fwics above?